### DISCUSSION TITLE
<title> I am approved to the github pro, but next step is not available now.

### DISCUSSION THREAD

**Discussion by wooseo-k** [community/community#111898] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![스크린샷 2024-03-10 195957](https://github.com/community/community/assets/162778500/90873f00-8068-4974-af48-f80fdb4876f6)

**Comment by wooseo-k** [comment]
Just they say benefits will be available soon, but after then, 5 hours passed by.

**Comment by SebastienXia** [comment]
Hello @wooseo-k 
I advise you to read at the very bottom of the page of this [link](https://github.com/orgs/community/discussions/111352#discussion-6342154), all the information you need is there!

**Reply by wooseo-k** [reply]
thank you very much for your help!!
